docs/written/trove/lists.scrbl

```diff
@@ -1623,13 +1623,104 @@ end
 }
 
 @list-method["member"]
+
+Returns true if the @pyret{elt} is located somewhere in the list.
+
+@examples{
+check:
+  [list: 'a', 'b', 'c'].member('a') is true
```

Comment:
Agreed.  Using `satisfies` with a curried method call seems needlessly tricky for an example meant to illustrate how the method works.